### PR TITLE
Debezium moving to Commonhaus Announcement

### PR DESCRIPTION
**Pull request by Naros**

### CODE REVIEW

File: _posts/2024-11-04-debezium-moving-to-commonhaus.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
++++<!-- more -->+++
+
+Today, we are thrilled to announce that the Debezium project is starting the transition to the Commonhaus Foundation.
+Debezium is the leading open-source distributed platform for Change Data Capture (CDC), and has been sponsored exclusively by Red Hat since its inception back in 2015.
+The move to CommonHaus aligns with our link:2024-10-03-plan-to-move-debezium-to-foundation[goal to foster a more inclusive, collaborative environment].
+
+== Why Move to a Foundation?
+
+Debezium has seen exceptional growth and adoption since 2025.
```
Comment (vjuranek) [inline _posts/2024-11-04-debezium-moving-to-commonhaus.adoc]: "2025" -> "2015"

**Review by vjuranek — COMMENTED** [review]
One minor thing, otherwise LGTM.

File: _posts/2024-11-04-debezium-moving-to-commonhaus.adoc (hunk 1 above)
Comment (ani-sha) [inline _posts/2024-11-04-debezium-moving-to-commonhaus.adoc]:
```suggestion
Debezium has seen exceptional growth and adoption since 2015.
```

**Comment by mfvitale** [comment]
LGTM!

**Pull request merged**